src/main/java/org/mule/extension/http/api/HttpRequestAttributes.java

```diff
@@ -130,4 +132,47 @@ public Certificate getClientCertificate() {
     return clientCertificate;
   }
 
+  public String toString() {
+    StringBuilder builder = new StringBuilder();
+    String tab = "   ";
+    String doubleTab = tab + tab;
+    builder.append(this.getClass().getName()).append(lineSeparator()).append("{").append(lineSeparator())
+        .append(tab).append("Request path=").append(requestPath).append(lineSeparator())
+        .append(tab).append("Method=").append(method).append(lineSeparator())
+        .append(tab).append("Listener path=").append(this.listenerPath).append(lineSeparator())
+        .append(tab).append("Local Address=").append(localAddress).append(lineSeparator())
+        .append(tab).append("Query String=").append(this.queryString).append(lineSeparator())
+        .append(tab).append("Relative Path=").append(this.relativePath).append(lineSeparator())
+        .append(tab).append("Remote Address=").append(this.remoteAddress).append(lineSeparator())
+        .append(tab).append("Request Uri=").append(this.requestUri).append(lineSeparator())
+        .append(tab).append("Scheme=").append(scheme).append(lineSeparator())
+        .append(tab).append("Version=").append(this.version).append(lineSeparator())
+        .append(tab).append("Headers=[").append(lineSeparator());
+
+    headers.entrySet().stream()
```

Comment:
If these maps are `MultiMap` then `entrySet` will mean repeated items will not appear. Use `entryList` in that case.